### ISSUE TITLE
Styling Statuses and Actions for Accessibility

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-reporting#562]
#### Describe the task
The current status, actions, and more actions styling do not pass the accessibility test
Users accidentally click on the status due to the pill styling, because it appears like a clickable button
Users do not intuitively click the Actions, because they don't realize it is clickable, and opt for clicking the status pills instead 
The more actions column is not accessible to screen readers users because "more" is not a descriptive enough table header. 

#### Acceptance Criteria
Change the status and actions styling as outlined in the [design guidelines](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=32645-10258&t=aRMEOnyZQaIrarnS-4)
- [ ] Remove the status pill outline 
- [ ] Add a button outline to the button text
- [ ] Rename the "more" column to "More Actions" 

Tables affected by this change
Registration
- Operators 
- Operations
- Facilities 
- Contacts
- User Access Grid

Reporting
- Annual reports
- Past submissions
- Report Information (facilities reports grid)
#### Definition of Done

- [ ] first
- [ ] second
- [ ] third

#### Additional context

- Add any other context about the task here.
- Or here

**Comment by zoeyli-46** [comment]
@patriciarussellCAS @shon-button ticket created for changing status and actions button/pill styling
